### PR TITLE
gsoc/2019: add GoHEPgroot dframe proposal

### PR DESCRIPTION
**Pull request by sbinet**
needs #444.

### CODE REVIEW

**Comment by sbinet** [comment]
@sebgadrat PTAL.

**Comment by sebgadrat** [comment]
LGTM too.

File: _gsocorgs/2019/cc-in2p3.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: "Centre de Calcul de l’IN2P3"
+author: "Sebastien Gadrat"
+layout: default
+organization: CC-IN2P3
+logo: logo_ccin2p3.png
+description: |
+  IN2P3’s Computing Centre (or CC-IN2P3) is a CNRS support and research unit linked to IN2P3,
+  the institute that pursues and coordinates research on particle physics, nuclear physics
+  and astroparticle physics. A national research infrastructure, the CC-IN2P3 designs
```
[the remark this answers is not in the document]
Comment (sbinet) [reply _gsocorgs/2019/cc-in2p3.md]: this indeed reads a bit funny, but we've just copy-pasted that blurb from their english website, so...

[the remark this answers is not in the document]
Comment (sebgadrat) [reply _gsocorgs/2019/cc-in2p3.md]: Yup, this reads a bit weird... I made a PR to update the file, with, I hope, some better english.

**Pull request merged**